docs/content/latest/yugabyte-cloud/develop/hasuracloud.md

```diff
@@ -0,0 +1,215 @@
+
+1. In the Hasura Cloud console, click the **Data** tab.
+
+1. Under **Connect Existing Database**, enter the following details.
+
+    * **Database Display Name**: yb-realtime-polldb
+    * **Data Source Driver**: PostgreSQL
+    * **Connect Database Via**: Database URL
+
+1. Set the **Database URL** using the password, IP address, and port from your Yugabyte Cloud cluster connection info, recorded earlier.
+
+    The Database URL is in the form `postgresql://admin:password@ip_address:port/yugabyte`.
+
+    For example, `postgresql://admin:59jh2hj@35.217.31.152:21301/yugabyte`.<br><br>
+
+    ![Connect Hasura database](/images/deploy/yugabyte-cloud/hasura-cloud-connect-database.png)<br><br>
+
+1. Click **Connect Database** and wait for confirmation that the database has connected.
+
+1. Click **View Database**. The schema will be empty.
+
+## Set up Realtime Poll and apply migrations
+
+On your machine, navigate to the `hasura` directory in the sample application directory.
+
+```sh
+$ cd realtime-poll/hasura
+```
+
+Edit the `config.yaml` file by changing the following parameters:
+
+* set `endpoint` to `https://yb-realtime-poll.hasura.app`.
+* set `admin_secret` to the **Admin Secret** you copied from the Hasura Cloud project dashboard.
+
+Navigate to the `migrations` directory in the `hasura` directory.
+
+```sh
+$ cd migrations
+```
+
+Rename the `default` directory to the Database Display Name you assigned to your Yugabyte Cloud database in the Hasura project console (that is, `yb-realtime-polldb`).
+
+```sh
+$ mv default yb-realtime-polldb
+```
+
+Return to the hasura directory and apply the migration.
+
+```sh
+$ cd ..
+$ hasura migrate apply --database-name yb-realtime-polldb
+```
+
+This creates the tables and views required for the polling application in the database.
+
+## Configure the Hasura project
+
+Return to the **Data** tab in the Hasura Cloud project console. The tables from your application are now listed under **Untracked tables or views**.
+
+Click **Track All** and **OK** to confirm to allow the tables to be exposed over the GraphQL API.
+
+Once the console refreshes, **Untracked foreign-key relationships** lists the relationships.
+
+Click **Track All** and **OK** to confirm to allow the relationships to be exposed over the GraphQL API.
+
+Add a new Array relationship for the `poll_results` table as follows:
+
+1. Select the `poll_results` table.
+
+1. Select the **Relationships** tab.
+
+1. Under **Add a new relationship manually**, click **Configure**.
+
+1. Set the following options:
+
+    * Set **Relationship Type** to Array Relationship.
+    * In the **Relationship Name** field, enter option.
+    * Set **Reference Schema** to public.
+    * Set **Reference Table** to option.
+    * Set **From** to `poll_id` and **To** to `poll_id`.<br><br>
+
+    ![Connect Hasura database](/images/deploy/yugabyte-cloud/hasura-cloud-relationships-add.png)<br><br>
+
+1. Click **Save**.
+
+## Configure Realtime Poll 
```

Comment:
Seems like this step would be better if it came after (or was part of) the "Set up Realtime Poll and apply migrations" section.

Reply:
Then you could call that section "Set up and configure Realtime Poll"